.circleci/config.yml

```diff
@@ -19,36 +18,43 @@ jobs:
           name: Install dependencies
           command: |
             # Install Terraform
-            wget -q "https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip"
-            unzip terraform_${TERRAFORM_VERSION}_linux_amd64.zip
+            sudo wget -qO /tmp/terraform_${TERRAFORM_VERSION}_linux_amd64.zip "https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip" && \
+            sudo unzip -d /usr/local/bin /tmp/terraform_${TERRAFORM_VERSION}_linux_amd64.zip && \
+            sudo chmod +x /usr/local/bin/terraform
 
             # Install Sops
             sudo wget -q "https://github.com/mozilla/sops/releases/download/${SOPS_VERSION}/sops-${SOPS_VERSION}.linux" -O /usr/local/bin/sops
             sudo chmod +x /usr/local/bin/sops
 
+            # prefixout used in covaelnce for logging output of commands delivered
+            sudo wget -qO /tmp/prefixout_0.1.0_linux_amd64.zip "https://github.com/WhistleLabs/prefixout/releases/download/v0.1.0/prefixout_0.1.0_linux_amd64.zip" && \
+            sudo unzip -d /usr/local/bin /tmp/prefixout_0.1.0_linux_amd64.zip && \
+            sudo chmod +x /usr/local/bin/prefixout;
+
             # Install gem bundle
```

Comment:
## LGTM but a couple notes that could clean it up a bit. 

#### Warning in circle ci during `bundle install`
```
Warning: the running version of Bundler (1.16.6) is older than the version that created the lockfile (1.17.3). We suggest you upgrade to the latest version of Bundler by running `gem install bundler`.
Fetching gem metadata from https://rubygems.org/.......
solargraph (0.32.3) has dependency bundler (>= 1.17.2), which is unsatisfied by the current bundler version 1.16.6, so the dependency is being ignored
```

#### If you add something like the below you could avoid this error and install solargraph without issue in circleci: 
* Add an environment variable to set the bundler version that was used to lock the Gemfile so it doesn't warn on circleci. 

```bash
environment:
      BUNDLER_VERSION: 1.17.2
```

* Add a gem install of bundler to make sure your using the correct version needed. 

```bash
# Install bundler version in Gemlock before install
gem install bundler:${BUNDLER_VERSION}
```

Also you might consider updating to ruby 2.5.5 instead of 2.5.1 and updating all the gems as well. Check our current for reference: 
* [unifio/covalence/.circleci/config.yml](https://github.com/unifio/covalence/blob/36cb9d1c68e8bc245914fcb2587c7a097af0a222/.circleci/config.yml#L8)
* [bundler install avoiding error on solargraph dependency](https://github.com/unifio/covalence/blob/36cb9d1c68e8bc245914fcb2587c7a097af0a222/.circleci/config.yml#L33)
* [ruby version set in `.ruby-version`](https://github.com/unifio/covalence/blob/36cb9d1c68e8bc245914fcb2587c7a097af0a222/.ruby-version#L1)
* [Gemfile.lock](https://github.com/unifio/covalence/blob/master/Gemfile.lock)